packages/components/app/styles/components/dropdown.scss

```diff
@@ -0,0 +1,410 @@
+// see https://github.com/hashicorp/design-system-components/issues/132
+//
+//
+
+@use '../mixins/focus-ring' as *;
+
+$hds-dropdown-toggle-base-height: 36px;
+$hds-dropdown-toggle-border-radius: 5px;
+$hds-dropdown-toggle-border-width: 1px;
+
+
+
+// TOGGLE/ICON
+.hds-dropdown-toggle-icon {
+  align-items: center;
+  background-color: transparent;
+  border: $hds-dropdown-toggle-border-width solid transparent; // We need this to be transparent for a11y
+  border-radius: $hds-dropdown-toggle-border-radius;
+  box-sizing: border-box; // TODO https://github.com/hashicorp/design-system-components/issues/46
+  color: var(--token-color-foreground-strong);
+  display: flex;
+  height: $hds-dropdown-toggle-base-height;
+  isolation: isolate;
+  justify-content: center;
+  outline-style: solid; // used to avoid double outline+focus-ring in Safari (see https://github.com/hashicorp/design-system-components/issues/161#issuecomment-1031548656)
+  outline-color: transparent; // We need this to be transparent for a11y
+  padding: 1px;
+  position: relative;
+  min-width: $hds-dropdown-toggle-base-height;
+
+  &:hover,
+  &.is-hover {
+    background-color: var(--token-color-surface-interactive);
+    border-color: var(--token-color-border-strong);
+    cursor: pointer;
+  }
+
+  &:focus,
+  &.is-focus {
+    @include hds-focus-ring-with-pseudo-element(
+      $top: -1px,
+      $right: -1px,
+      $bottom: -1px,
+      $left: -1px,
+      $radius: $hds-dropdown-toggle-border-radius
+    );
+  }
+
+  &:active,
+  &.is-active {
+    background-color: var(--token-color-surface-interactive-active);
+    border-color: var(--token-color-border-strong);
+  }
+}
+
+.hds-dropdown-toggle-icon__wrapper {
+  align-items: center;
+  border-radius: 3px; // $hds-dropdown-toggle-border-radius- 1px padding - 1px border
+  display: flex;
+  justify-content: center;
+  height: 2rem;
+  padding: 1px;
+  width: 2rem;
+
+  img {
+    border-radius: inherit;
+  }
+}
+
+.hds-dropdown-toggle-icon__chevron {
+  margin-left: 0.25rem;
+}
+
+// TOGGLE/BUTTON
+.hds-dropdown-toggle--with-button-component {
+  box-shadow: none; // we override this to remove the elevation style
+
+  .hds-button__icon {
+    margin-right: -6px; // we apply a negative margin to counter the padding-right of the button and reduce the visual space between the icon and the right border
+    margin-left: 0.5rem; // this overrides the rule `.hds-button__text + .hds-button__icon`
+  }
+}
+
+
+// LIST
+// UL ELEMENT
+// GOES INSIDE HDS::DISCLOSURE's :content block
+
+.hds-dropdown-list {
+  background-color: var(--token-color-surface-primary);
+  border-radius: 6px;
+  box-shadow: var(--token-surface-high-box-shadow);
+  list-style: none;
+  margin: 0;
+  max-width: 25rem;
+  min-width: 12.5rem;
+  padding: 0.25rem 0;
+  &.hds-dropdown-list--position-right {
+    position: absolute;
+    right: 0;
+    top: calc(100% + 0.25rem);
+    z-index: 2; // https://github.com/hashicorp/design-system/issues/114
```

Comment:
[issue] this in theory should have a proper value, 2 is just a "random" number here. maybe look into Structure or Cloud UI and see what value of Z they are using for this, and let's use the same

Reply:
yes, that's why we created an issue to track it, remember?

Reply:
My understanding is that the issue is about another problem. 
The issue is about the standardization (this will involve the designers too, we will have specific design tokens, a documentation page, etc) 
What I am suggesting here it to make sure that the value we're using for the dropdown works (and using the existing Structure/Cloud UI is a good bet).

Reply:
I checked in Cloud UI and apparently their dropdown doesn't use a z-index (not sure how that works). I tried but in our case this would not work, we need a z-index. Let's leave it as is for now, and once we work on the adoption test we can  decide what is the right value to use here.